modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/common/LogInitialization.scala

```diff
@@ -8,7 +8,26 @@ import scalaz._
 import Scalaz._
 import scalaz.effect.IO
 
-trait LogInitialization {
+trait AppBaseDir {
+  /**
+    * Calculates the base dir of the application based on the location of "this" class jar file
+    * It will throw an exception if unable to find the base dir
+    */
+  def baseDir: File = {
```

Comment:
This is called from inside `IO` but I guess you want to make it explicit that is run inside `IO`?